### PR TITLE
[#5542][#5482] improvment(hadoop-catalog): Update Hadoop version in Hadoop catalog module from 3.1 to 3.3

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Change the version of hadoop common and hadoop client from 3.1.0 to 3.3.0

### Why are the changes needed?

3.1.0 is old and need to be update-to-date. 

Fix: #5542
Fix: #5482

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

Existing CI.

### CODE REVIEW

**Comment by jerryshao** [comment]
Please change the PR title to like "[#xxx][#xxx]". Also, please confirm if the LICENSE and NOTICE file should be updated.

File: catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopOSSCatalogIT.java [hunk 1]
```diff
@@ -197,4 +195,11 @@ public void testCreateSchemaAndFilesetWithSpecialLocation() {
     // Delete catalog
     metalake.dropCatalog(localCatalogName, true);
   }
+
+  protected static boolean ossIsPrepare() {
```
Comment (jerryshao) [inline catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopOSSCatalogIT.java:199]: You'd better using the same method name like "isXXXConfigured".

Comment (yuqi1129) [reply catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopOSSCatalogIT.java]: fixed.

File: clients/client-python/tests/integration/test_gvfs_with_gcs.py [hunk 2]
```diff
@@ -36,12 +36,21 @@
 logger = logging.getLogger(__name__)
 
 
-@unittest.skip("This test require GCS service account key file")
+def oss_is_prepare():
```
Comment (jerryshao) [inline clients/client-python/tests/integration/test_gvfs_with_gcs.py:39]: I think you'd better also unify the python method name, as well as disable messages, some places show "xxx is not prepared", some places show "xxx is not configured".

**Review by jerryshao — APPROVED** [review]

**Pull request merged**